app/controllers/invitations/psp/DeclarationController.scala

```diff
@@ -112,13 +112,13 @@ class DeclarationController @Inject()(
               }
             } recoverWith {
               case _: ActiveRelationshipExistsException =>
-                Future.successful(Redirect(controllers.invitations.psp.routes.AlreadyAssociatedWithSchemeController.onPageLoad()))
+                Future.successful(Redirect(controllers.invitations.psp.routes.AlreadyAssociatedWithSchemeController.onPageLoad))
             }
           case _ =>
-            Future.successful(Redirect(controllers.routes.SessionExpiredController.onPageLoad()))
+            Future.successful(Redirect(controllers.routes.SessionExpiredController.onPageLoad))
         }
     }.left.map(_ =>
```

Comment:
Comment only: do we still need to project with left before mapping? Does our Either logic assume Right if not explicitly stated? Just trying to understand. Will not repeat comment for any future usages of .left

Reply:
I think Either is right-biased, which means that Right is assumed to be the default case to operate on @cyeniceri @grantrobson could you confirm please?

Reply:
Errors are usually the Left of an Either and the happy case is usually the Right. So yes, I think you are right

Reply:
yes, you don't need to say .right explicitly. It's default right anymore. Errors are on left as Grant said.